lighthouse-treemap/app/src/main.js

```diff
@@ -161,10 +161,14 @@ class TreemapViewer {
       this.updateColors();
     });
 
-    treemapEl.addEventListener('keypress', (e) => {
+    treemapEl.addEventListener('keyup', (e) => {
       if (!(e instanceof KeyboardEvent)) return;
 
       if (e.key === 'Enter') this.updateColors();
+
+      if (e.key === 'Escape' && this.treemap) {
+        this.treemap.zoom([]); // zoom out to root
```

Comment:
is there potential for this have the same recoloring problem (https://github.com/GoogleChrome/lighthouse/pull/12496#issuecomment-842706302)? do we need to flip the order to ensure these are recolored? From what I could tell poking around devtools it seemed like the other nodes were just hidden and still there, so not an issue, but calling it out in case it was just oversight.

Reply:
played around with it. zoomed into a node, changed view mode with keyboard, zoomed out with ESC. things were colored correctly.